### DISCUSSION TITLE
GreptimeDB 能力咨询

### DISCUSSION THREAD

**Discussion by Jeremy-boo** [GreptimeTeam/greptimedb#4612] [Q&A] [answered]
按照 GreptimeDB 简介中所说, 现在 GreptimeDB 能同时处理或者存储指标（Metrics）、日志（Logs）、事件（Events）和追踪（Traces）这四种类型的数据吗

**Comment by killme2008** [comment, marked as answer]
Hello，感谢关注。目前 metrics, logs 和 events 都是支持的， trace 我们还在计划中，预计要到年底或者明年初左右。
